src/dirtyfields/compare.py

```diff
@@ -0,0 +1,38 @@
+import datetime
+import pytz
+import warnings
+
+from django.conf import settings
+from django.utils import timezone
+
+
+def raw_compare(cls, new_value, old_value):
+    return new_value == old_value
+
+
+def time_zone_support_compare(cls, new_value, old_value):
+
+    if isinstance(new_value, datetime.datetime) and isinstance(old_value, datetime.datetime):
```

Comment:
Cosmetics again: We can remove a level of indentation again by returning early "when one of the instances isn't a datetime". Remember the zen of Python: "Flat is better than nested."